### ISSUE TITLE
Python version in .pre-commit-config.yaml

### ISSUE THREAD

**Issue by cmb87** [cmb87/TurbofanDesign#2]
Which Python version are we using? Need to manually adjust Python version from 3.7 to 3.6. I guess it is the other way around for you, Matthias :)? 
Proposal 1: Let's define one version
Proposal2: Is it possible to specify 3.7 and 3.6 in the yaml file?

**Comment by matzehuels** [comment]
Yes, I saw this issue. Fix is to just get rid of "language_version", which I did, so it should be running for any python version now! 

I still think you should upgrade to python 3.7 hehe, but I dont see any urgency to do so, as we are not using any py3.7 specific stuff as far as I can see.
